### DISCUSSION TITLE
%E6%8B%93%E5%B1%95%E6%8A%80%E6%9C%AF/%E6%90%AD%E5%BB%BA%E8%BF%90%E8%A1%8C%E7%8E%AF%E5%A2%83/vscode%E8%B0%83%E8%AF%95rmmv%E9%A1%B9%E7%9B%AE

### DISCUSSION THREAD

**Discussion by giscus** [RuanZhongNan/RPGMV-dev-notes#136] [Announcements]
# %E6%8B%93%E5%B1%95%E6%8A%80%E6%9C%AF/%E6%90%AD%E5%BB%BA%E8%BF%90%E8%A1%8C%E7%8E%AF%E5%A2%83/vscode%E8%B0%83%E8%AF%95rmmv%E9%A1%B9%E7%9B%AE

用 VScode 的方式来运行调试 mv 项目 本小节内容涉及到了插件开发的内容，但是并不涉及具体代码编写，只涉及到简单的软件设置。 在实际开发的时候，会遇到这样的一个开发情况：我需要先对插件设置断点，然后再启动游戏触碰断点，随后再跟着进程一步一步的摸清楚代码的运行情况。虽然 RPGMV 本身会提供一个调试工具，按 F8 可以开启调试工具，但是这个工具...

https://www.ruan-cat.com/%E6%8B%93%E5%B1%95%E6%8A%80%E6%9C%AF/%E6%90%AD%E5%BB%BA%E8%BF%90%E8%A1%8C%E7%8E%AF%E5%A2%83/vscode%E8%B0%83%E8%AF%95rmmv%E9%A1%B9%E7%9B%AE.html

**Comment by Flow-sandyu** [comment]
这种情况下启动Live Server后，并不能读取save文件夹的存档啊，有方法解决这个吗
